src/azure-cli/azure/cli/command_modules/rdbms/_util.py

```diff
@@ -84,3 +85,21 @@ def create_firewall_rule(cmd, resource_group_name, server_name, start_ip, end_ip
     firewall = firewall_client.begin_create_or_update(resource_group_name, server_name, firewall_name, parameters)
 
     return firewall.result().name
+
+
+def retryable_method(retries=3, interval_sec=5, exception_type=Exception, condition=None):
```

Comment:
We have retry decorator in cli core: https://github.com/Azure/azure-cli/blob/7b12d40626bd52f60ef601c727d5d8b9433bc886/src/azure-cli-core/azure/cli/core/decorators.py#L89-L113

Maybe we can add `condition` to cli core retry decorator in the future. @jiasli for awareness

Reply:
Yeah, I noticed that decorator before, but I want to check a specific property of the exception to know if it's retryable or not.